apps/api/src/auth/auth.module.ts

```diff
@@ -21,7 +22,8 @@ import { GitlabStrategy } from '../config/oauth-strategy/gitlab/gitlab.strategy'
         algorithm: 'HS256'
       }
     }),
-    UserModule
+    UserModule,
+    CacheModule
```

Comment:
Since CacheModule is globally exported, I would ask you to make its import in `app.module.ts`. That way, all the other modules will have access to it.